terraform/modules/concourse_web/templates/web_bootstrap.sh

```diff
@@ -15,6 +15,11 @@ aws ssm get-parameter --with-decryption --name ${session_signing_key_ssm_id} | j
 aws ssm get-parameter --with-decryption --name ${tsa_host_key_ssm_id} | jq -r .Parameter.Value > /etc/concourse/host_key
 aws ssm get-parameter --with-decryption --name ${authorized_worker_keys_ssm_id} | jq -r .Parameter.Value > /etc/concourse/authorized_worker_keys
 
+%{ for cert in ${enterprise_github_certs} }
```

Comment:
this, however, does not plan:
```../modules/concourse_web/templates/web_bootstrap.sh:18,16-17: Invalid
character; This character is not used within the language., and 1 other
diagnostic(s)```
I'll investigate

Reply:
I feel this is because there's a terraform for loop, trying to run inside a bash script? hence: `This character is not used within the language.`

Reply:
Fixed.